R/last.R

```diff
@@ -9,14 +9,16 @@ last = function(x, ...) {
       if (!length(x)) return(x) else return(x[[length(x)]])  # for vectors, [[ works like [
     } else if (is.data.frame(x)) return(x[NROW(x),])
   }
+  # nocov start
```

Comment:
I saw the `if` branches covered in `first` but not `last` (despite apparently neither actually being hit in tests), so just throw the whole sections in `nocov` to be sure. maybe surface to Jim as well...